CHANGELOG.md

```diff
@@ -1,9 +1,31 @@
+<!-- markdownlint-disable MD012 no-multiple-blanks -->
 <!-- markdownlint-disable MD024 no-duplicate-heading/no-duplicate-header -->
 
 # BuildMasterAutomation Changelog
 
+## 3.0.2
+
+> Released 17 Jul 2023
+
+### Changes
+
+* `Get-BMEnvironment` ensures that returned objects always have a `parentName` property.
+
+### Fixed
+
+* Some expected/handled errors are not ignored.
+* `Get-BMRaftItem` searches *all* applications when looking for a global item.

```

Comment:
Can't tell what was fixed. Was it behaving in this way incorrectly or was it supposed to behave in this way but wasn't?